src/tracks/InstrumentTrack.cpp

```diff
@@ -1066,7 +1066,8 @@ InstrumentTrackView::InstrumentTrackView( InstrumentTrack * _it, TrackContainerV
 
 InstrumentTrackView::~InstrumentTrackView()
 {
-	freeInstrumentTrackWindow();
+	delete m_window;
```

Comment:
What about using `unique_ptr` instead?

Reply:
I think it's out of scope, but it's doable.